packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx

```diff
@@ -231,6 +258,13 @@ const Header = ({ store, signals, zenMode }: Props) => {
           store={store}
         />
 
+        <ShareButton
```

Comment:
Ahh, I've probably been confusing with the design! We should have this button before the "Fork" button. It changed later in the designs 😄

Reply:
Yeah I thought you wanted to move the position of the button 👍 anyway now buttons are there as they were before.